src/dependency-manager/src/service-config/v1.ts

```diff
@@ -253,6 +253,14 @@ export class ServiceConfigV1 extends ServiceConfig {
   @Type(() => BuildSpecV1)
   build?: BuildSpecV1;
 
+  @IsOptional({ always: true })
+  @IsNumber()
+  cpu?: number;
```

Comment:
Everything needs to be a string because of interpolation

ex
`cpu: ${{ parameters.cpu }}`

replace IsNumber with `@Type(() => String)`
Probably should do the same for memory